terraform/modules/vault/files/config.hcl

```diff
@@ -46,3 +46,5 @@ ha_storage "dynamodb" {
   read_capacity  = "5"
   write_capacity = "5"
 }
+
+${ vault_additional_config }
```

Comment:
Terraform has [directives](https://www.terraform.io/docs/configuration/expressions.html#directives) (code inside templates) now which you could add here if you're interested, although it's not really necessary.

For example
```hcl
%{ ifvault_additional_config != "" }
${ vault_additional_config }
%{ endif }
```

Reply:
Neat, it's up to you!